sound/soc/intel/boards/sof_sdw.c

```diff
@@ -863,8 +863,10 @@ static int sof_card_dai_links_create(struct device *dev,
 	for (i = 0; i < ARRAY_SIZE(codec_info_list); i++)
 		codec_info_list[i].amp_num = 0;
 
-	hdmi_num = sof_sdw_quirk & SOF_SDW_TGL_HDMI ?
-				SOF_TGL_HDMI_COUNT : SOF_PRE_TGL_HDMI_COUNT;
+	if (sof_sdw_quirk & SOF_SDW_TGL_HDMI)
+		hdmi_num = SOF_TGL_HDMI_COUNT;
+	else
+		hdmi_num = SOF_PRE_TGL_HDMI_COUNT;
```

Comment:
just grepped the sound/ and sound/soc trees for "\?.*:" and got 2601 and 835 hits respectively, so, even if Mark Brown doesn't particularly like them, doesn't look like he's enforcing that.

Reply:
https://mailman.alsa-project.org/pipermail/alsa-devel/2020-August/172308.html

There's no point in arguing.